ariadne/cardano/src/Ariadne/MainTemplate.hs

```diff
@@ -170,14 +175,15 @@ initializeEverything MainSettings {..}
     mainAction :: IO ()
     mainAction = do
       initAction
-
+      withAsync postInitAction $ \postInitActionThread -> do
+        linkUI postInitActionThread $ msPutBackendErrorToUI uiFace
       -- Spawn backend actions in async thread, then run ui action in the main thread
       -- This is needed because some UI libraries (Qt) insist on livng in the main thread
-      withAsync serviceAction $ \serviceThread -> do
-        -- Make custom link to service thread for UI apps.
-        -- It is going to call msPutBackendErrorToUI and rethrow exception, if something goes wrong
-        linkUI serviceThread $ msPutBackendErrorToUI uiFace
-        uiAction
+        withAsync serviceAction $ \serviceThread -> do
```

Comment:
Indentation is mismatched between this line and the previous line. I'm not certain why this block has to be indented 2 spaces further, but it doesn't seem to cause any bugs. Anyway, please either indent the comment or dedent the block.